rmw_dds_common/src/security.cpp

```diff
@@ -35,7 +35,7 @@ bool get_security_files(
     {"PERMISSIONS", "permissions.p7s"},
   };
 
-  for (const std::pair<std::string, std::string> & el : required_files) {
+  for (const auto & el : required_files) {
```

Comment:
I'm not sure how I missed this when I merged in #51, so sorry about that.  That said, I'd really prefer to make this:

```suggestion
  for (const std::pair<const std::string, std::string> & el : required_files) {
```

Reply:
that's fine for me, but just out of curiosity. Is there any specific reason why you think the auto deduction doesn't cut it?

Reply:
> that's fine for me, but just out of curiosity. Is there any specific reason why you think the auto deduction doesn't cut it?

`auto` will clearly work.  I just dislike `auto` in general, as it makes reading the code much harder.